pkg/verification/service/verification_service_test.go

```diff
@@ -375,27 +336,13 @@ func (s *TestVerificationServiceSuite) TestInitVerificationPassesWhenMaxCountRea
 
 	gock.Observe(obs)
 
-	userSignup := &toolchainv1alpha1.UserSignup{
-		TypeMeta: metav1.TypeMeta{},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "123",
-			Namespace: configuration.Namespace(),
-			Annotations: map[string]string{
-				toolchainv1alpha1.UserSignupVerificationInitTimestampAnnotationKey: now.Add(-25 * time.Hour).Format(verificationservice.TimestampLayout),
-				toolchainv1alpha1.UserVerificationAttemptsAnnotationKey:            "3",
-				toolchainv1alpha1.UserSignupVerificationCodeAnnotationKey:          "123456",
-			},
-			Labels: map[string]string{
-				toolchainv1alpha1.UserSignupUserPhoneHashLabelKey: "+1NUMBER",
-			},
-		},
-		Spec: toolchainv1alpha1.UserSignupSpec{
-			IdentityClaims: toolchainv1alpha1.IdentityClaimsEmbedded{
-				PreferredUsername: "shane@redhat.com",
-			},
-		},
-	}
-	states.SetVerificationRequired(userSignup, true)
+	userSignup := testusersignup.NewUserSignup(
+		testusersignup.WithName("johny"),
+		testusersignup.WithLabel(toolchainv1alpha1.UserSignupUserPhoneHashLabelKey, "+1NUMBER"),
+		testusersignup.WithAnnotation(toolchainv1alpha1.UserSignupVerificationInitTimestampAnnotationKey, now.Format(verificationservice.TimestampLayout)),
```

Comment:
The old code has `now.Add(-25 * time.Hour).Format(verificationservice.TimestampLayout)`. Is it ok to move the timestamp 25 hours forward? I don't understand the code at all, just observing the difference...

Reply:
good catch, thanks. There was apparently also a bug in the actual prod code, fixed in https://github.com/codeready-toolchain/registration-service/commit/7891733c3928a90de88ac0feb0236057dc9c6f76